### PR TITLE
Update README.md to add a section on M1 support

### PR DESCRIPTION
**Pull request by hokitam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,14 @@ datastreams.Start(datastreams.WithAgentAddr("notlocalhost:8126"))
 ```
 
 The instrumentation relies on creating checkpoints at various points in your data stream services, recording the pathway that messages take along the way. These pathways are stored within a Go Context, and are passed around via message headers.
- 
+
+### M1 support
```
Comment (piochelepiotr) [inline README.md:33]: can we change the title to `M1 support of the Kafka instrumentation`

File: README.md [hunk 2]
```diff
@@ -29,7 +29,14 @@ datastreams.Start(datastreams.WithAgentAddr("notlocalhost:8126"))
...
+
+On M1 machines, the following parameters are needed when starting the instrumented service:
```
Comment (piochelepiotr) [inline README.md:35]: Just a little bit of context: This library relies on [Confluent Kafka Go](https://github.com/confluentinc/confluent-kafka-go)

**Review by piochelepiotr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hokitam) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (hokitam) [reply README.md]: done

**Pull request merged**